doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
```

Comment:
Strong no, at all yes.
But if there is no good reason to have something, why have it?